standalone/src/models/store/store/fields-config.ts

```diff
@@ -0,0 +1,78 @@
+import { fake, oneOf, sequence, type TModelFieldsConfig } from '@/core';
+import { Channel } from '@/models/channel';
+import {
+  ClientLogging,
+  LocalizedString,
+  ReferenceGraphql,
+  TLocalizedStringGraphql,
+} from '@/models/commons';
+import { createRelatedDates } from '@/utils';
+import { TStoreRest, TStoreGraphql } from './types';
+
+const [getOlderDate, getNewerDate] = createRelatedDates();
+
+export const restFieldsConfig: TModelFieldsConfig<TStoreRest> = {
+  fields: {
+    id: fake((f) => f.string.uuid()),
+    version: sequence(),
+    key: fake((f) => f.lorem.slug()),
+    name: fake(() => LocalizedString.random()),
+    languages: [oneOf('en-US', 'de-DE', 'es-ES')],
+    countries: [fake((f) => ({ code: f.location.countryCode() }))],
+    distributionChannels: [fake(() => Channel.ChannelRest.random())],
+    supplyChannels: [fake(() => Channel.ChannelRest.random())],
+    createdAt: fake(getOlderDate),
+    lastModifiedAt: fake(getNewerDate),
+    lastModifiedBy: fake(() => ClientLogging.random()),
+    createdBy: fake(() => ClientLogging.random()),
+    productSelections: [],
+    custom: null,
+  },
+};
+
+export const graphqlFieldsConfig: TModelFieldsConfig<TStoreGraphql> = {
+  fields: {
+    __typename: 'Store',
+    id: fake((f) => f.string.uuid()),
+    version: sequence(),
+    key: fake((f) => f.lorem.slug()),
+    name: fake(() => LocalizedString.random()),
+    languages: [oneOf('en-US', 'de-DE', 'es-ES')],
+    countries: [fake((f) => ({ code: f.location.countryCode() }))],
+    productSelections: [],
+    custom: null,
+    createdAt: fake(getOlderDate),
+    lastModifiedAt: fake(getNewerDate),
+    createdBy: fake(() => ClientLogging.random()),
+    lastModifiedBy: fake(() => ClientLogging.random()),
+    distributionChannels: [fake(() => Channel.ChannelRest.random())],
+    supplyChannels: [fake(() => Channel.ChannelRest.random())],
+    distributionChannelsRef: null,
+    supplyChannelsRef: null,
+    nameAllLocales: null,
+  },
+  postBuild: (model) => {
+    const nameAllLocales = model.name as unknown as TLocalizedStringGraphql;
```

Comment:
This should be done the other way around. 
We should populate the `nameAllLocales` property in the `fields` section and derive the `name` property in the `postBuild` function.
[Here](https://github.com/commercetools/test-data/blob/main/standalone/src/models/cart/cart/line-item/field-config.ts#L81) is an example.